sqe-api-server/HttpControllers/SignInterpretationController.cs

```diff
@@ -146,13 +152,14 @@ public async Task<ActionResult<AttributeDTO>> CreateEditionSignInterpretationAtt
         }
 
         /// <summary>
-        /// Links two sign interpretations in the edition's sign stream 
+        ///     Links two sign interpretations in the edition's sign stream
```

Comment:
Yeah, this frustrates me a lot.  I think if I integrated a more heavy handed "linter", it would fix this.  I do have a "linter" of sorts that runs before every push, but perhaps I can make it even more rigid so we don't get this noise.  My apologies.